### ISSUE TITLE
no shortcut in the safari dock shortcut app

### ISSUE THREAD

**Issue by cxsatt** [arnoappenzeller/PiPifier#100]
since the new macOS sequoia there is support for extension in the safari "add to dock" apps.
there are several apps that already support for it but there is none for pip at the moment.
pls pls make it so that there can be one in the dock apps

**Comment by wwessex** [comment]
Yes please tweak PiPifier to work in docked websites, as I have YouTube docked as its own app and Pip does not work
